docs/examples/getDataMoverResources.go

```diff
@@ -0,0 +1,318 @@
+	exitOnError(err)
+}
+
+func getClusterResourceItems(resourceSpecification schema.GroupVersionResource, namespace string) []unstructured.Unstructured {
+	resource, err := DynamicClient.Resource(resourceSpecification).Namespace(namespace).List(context.TODO(), metaV1.ListOptions{})
+	exitOnError(err)
+	return resource.Items
+}
+
+func getPodForDeployment(deployment *appsV1.Deployment) coreV1.Pod {
+	listOptions := metaV1.ListOptions{LabelSelector: labels.Set(deployment.Spec.Selector.MatchLabels).AsSelector().String()}
+	pods, err := KubernetesClient.CoreV1().Pods(namespace).List(context.TODO(), listOptions)
+	exitOnError(err)
+	if len(pods.Items) > 1 {
+		fmt.Printf("More then one Pod (%v) for deployment %v\n", pods.Items, deployment)
+		os.Exit(1)
+	}
+	return pods.Items[0]
+}
+
+func execCommandInVeleroPod(command string) *bytes.Buffer {
+	deployment, err := KubernetesClient.AppsV1().Deployments(namespace).Get(context.TODO(), "velero", metaV1.GetOptions{})
+	exitOnError(err)
+
+	option := &coreV1.PodExecOptions{
+		Command:   strings.Split(command, " "),
+		Stdin:     false,
+		Stdout:    true,
+		Stderr:    true,
+		TTY:       true,
+		Container: "velero",
+	}
+	request := KubernetesClient.CoreV1().RESTClient().Post().Resource("pods").
+		Name(getPodForDeployment(deployment).Name).Namespace(namespace).SubResource("exec").
+		VersionedParams(option, scheme.ParameterCodec)
+
+	exec, err := remotecommand.NewSPDYExecutor(ClusterConfig, "POST", request.URL())
+	exitOnError(err)
+
+	output := &bytes.Buffer{}
+	errOutput := &bytes.Buffer{}
+
+	err = exec.StreamWithContext(context.TODO(), remotecommand.StreamOptions{Stdout: output, Stderr: errOutput})
+	if err != nil {
+		fmt.Println(output)
+		fmt.Println(errOutput)
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	return output
+}
+
+func backupSummary() {
+	backups := execCommandInVeleroPod("./velero get backup")
+	fmt.Printf("Get Backups:\n%v\n", backups)
+
+	volumeSnapshots := getClusterResourceItems(VolumeSnapshotResource, "")
+	fmt.Printf("Total Snapshots: %v\n", len(volumeSnapshots))
+	namespaceVolumeSnapshots := getClusterResourceItems(VolumeSnapshotResource, namespace)
+	fmt.Printf("Total OADP Snapshots: %v\n", len(namespaceVolumeSnapshots))
+	volumeSnapshotContents := getClusterResourceItems(VolumeSnapshotContentResource, "")
+	fmt.Printf("Total SnapshotContents: %v\n\n", len(volumeSnapshotContents))
+
+	volumeSnapshotBackups := getClusterResourceItems(VolumeSnapshotBackupResource, "")
+	phaseCompleted := 0
+	phaseInProgress := 0
+	phaseSnapshotBackupDone := 0
+
+	batchingCompleted := 0
+	batchingProcessing := 0
+	batchingQueued := 0
+	increaseBatching := map[string]func(){
+		"Completed":  func() { batchingCompleted += 1 },
+		"Processing": func() { batchingProcessing += 1 },
+		"Queued":     func() { batchingQueued += 1 },
+	}
+	for _, volumeSnapshotBackup := range volumeSnapshotBackups {
+		// TODO there is an easier way to access the properties?
```

Comment:
is there a less verbose way of getting properties from object?

Reply:
what do you mean less verbose?

Reply:
something like `volumeSnapshotBackup.Object["status"]["phase"]` instead of `volumeSnapshotBackup.Object["status"].(map[string]interface{})["phase"]`